data/ecosystems/p/polkadot.toml

```diff
@@ -161,6 +165,7 @@ sub_ecosystems = [
   "Tea Project",
   "Ternoa",
   "Tidal Finance",
+  "Tide Labs",
```

Comment:
Tide Labs shouldn't be part of Polkadot if they are just using substrate.  It is similar to chains using Tendermint.